src/app/services/config.ts

```diff
@@ -1,10 +1,16 @@
 export const changellyNodeApiEndPoints = {
   getCurrencies: 'getCurrencies',
-  getMinExchange: 'getMinAmount',
-  getEstimatedExchange: 'getExchangeAmount',
+  getMinAmount: 'getMinAmount',
+  getExchangeAmount: 'getExchangeAmount',
   getGenerateAddress: 'generateAddress',
   getTransaction: 'getTransactions',
   getExchangeStatus: 'getStatus',
 }
 
 export const nodeApiBaseUrl = 'http://localhost:3000/api/'
+
+export const changellyConstData = {
+  'CHANGELLY_FEE': 0.995,
```

Comment:
Changelly commission
There is no secret that Changelly charges 0.5% commission for each transaction. It’s charged after an exchange process is finished, but before your money is sent to a recipient’s address.